ios/FluentUI/Vnext/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+// MARK: - State
+
+public class PopupMenuItemCellState: MSFListCellState {
+
+	// Set/Unset selection for item cell
+	@objc @Published public var isSelected: Bool = false {
+		didSet {
+			if isSelected {
+				leadingUIView?.tintColor = cellTokens.selectionColor
+				tokens.labelColor = cellTokens.selectionColor
+				tokens.sublabelColor = cellTokens.selectionColor
+				tokens.trailingItemForegroundColor = cellTokens.selectionColor
+				accessoryType = .checkmark
+			} else {
+				leadingUIView?.tintColor = nil
+				tokens.labelColor = cellTokens.labelColor
+				tokens.sublabelColor = cellTokens.sublabelColor
+				tokens.trailingItemForegroundColor = cellTokens.trailingItemForegroundColor
+				accessoryType = .none
+			}
+		}
+	}
+	
+	// Disable selection for cell
+	@objc @Published public var isDisabled: Bool = false {
+		didSet {
+			if isDisabled {
+				leadingUIView?.tintColor = cellTokens.disabledColor
+				tokens.labelColor = cellTokens.disabledColor
+				tokens.sublabelColor = cellTokens.disabledColor
+				self.action = onTapAction
+				onTapAction = {}
+			} else {
+				leadingUIView?.tintColor = nil
+				tokens.labelColor = cellTokens.labelColor
+				tokens.sublabelColor = cellTokens.sublabelColor
+
+				// restore action
+				if self.action != nil {
+					self.onTapAction = self.action
+				}
+			}
+		}
+	}
+
+	var cellTokens: MSFPopupMenuItemCellTokens = MSFPopupMenuItemCellTokens()
+
+	private var action: (() -> Void)?
+}
+
+// MARK: - View
+
+public struct PopupMenuItemCell: View {
+    @Environment(\.theme) var theme: FluentUIStyle
+    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    @ObservedObject var state: PopupMenuItemCellState
+	@ObservedObject var tokens: MSFCellBaseTokens
```

Comment:
nit: indentation

Reply:
This is pretty strange, I have no idea why this indentation is showing up differently on github. 
I see the same in Xcode and VSCode
![Screen Shot 2021-06-10 at 10 25 38 AM](https://user-images.githubusercontent.com/63682282/121570033-50294b80-c9d6-11eb-983e-341fe131e1d6.png)

Reply:
Maybe it's the tabs? You can enable showing whitespace on Xcode to check